site/src/components/PaginationWidget/PaginationNavButton.tsx

```diff
@@ -0,0 +1,84 @@
+import { type ReactNode, useEffect, useState } from "react";
+
+import Button from "@mui/material/Button";
+import Tooltip from "@mui/material/Tooltip";
+import { useTheme } from "@emotion/react";
+
+type PaginationNavButtonProps = {
+  children: ReactNode;
+  disabled: boolean;
+  disabledMessage: ReactNode;
+  onClick: () => void;
+  "aria-label": string;
```

Comment:
Rather than special case this one prop, we should probably just extend `HTMLAttributes`

Reply:
I can extend it, but the thing is that even if I extend `HTMLAttributes` or `componentPropsWithoutRef`, I'll still need to treat `aria-label` as a special case to enforce that it's required

Reply:
eh, fair enough.